gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/client.py.j2

```diff
@@ -476,6 +485,22 @@ class {{ service.client_name }}(metaclass={{ service.client_name }}Meta):
         )
         {% endif %}
 
+{#
+    Add API Version to metadata as per https://github.com/aip-dev/google.aip.dev/pull/1331.
+    When using this macro, ensure the calling template
+    generates the following lines:
+        try:
+            from google.api_core import version_header
+            HAS_GOOGLE_API_CORE_VERSION_HEADER = True  # pragma: NO COVER
+        except ImportError:  # pragma: NO COVER
+            HAS_GOOGLE_API_CORE_VERSION_HEADER = False
+#}
+    {% if service.version %}
+        if HAS_GOOGLE_API_CORE_VERSION_HEADER:  # pragma: NO COVER
+            metadata = tuple(metadata) + (
+                version_header.to_api_version_header("{{ service.version }}"),
```

Comment:
I'm not seeing `to_api_version_header` in any of the golden files.....

Reply:
This is WAI. Showcase is the only API that supports API Versioning at the moment.
Filed https://github.com/googleapis/gapic-generator-python/issues/2022 to add showcase to the golden files.